.github/workflows/TEMPLATE_actor.yml

```diff
@@ -30,6 +32,8 @@ jobs:
       uses: wasmcloud/common-actions/rust-check@main
       with:
         working-directory: ${{ env.working-directory }}
+        # The `--doc` is required for wasm, as cargo cannot execute wasm tests by default
```

Comment:
I'd prefer to fix it a different way, but this is ok for now. Actor developers could put tests in their code, so we do want "make test" and "cargo test" to work. Since there aren't any tests in the actor code yet, your solution works for now.

Reply:
Yeah that makes sense, we can probably move the `cargo test` step outside of the rust check directory as some tests require some different arguments